code/game/objects/structures/crates_lockers/closets/secure/security.dm

```diff
@@ -57,7 +56,6 @@
 	new /obj/item/clothing/suit/armor/vest/alt(src)
 	new /obj/item/assembly/flash/handheld(src)
 	new /obj/item/clothing/glasses/sunglasses(src)
-	new /obj/item/restraints/handcuffs/cable/zipties(src)

```

Comment:
ties in hoplocker are very useful, hop gets tided a shitload.
In a way it encourages hopcurity, but dealing with shitheads that run in or climb over the desk is a pain.

Reply:
there are handcuffs in bridge